db/migrate/20220329203402_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :posts_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- You have done great fixing some of the changes and the requirements from the previous reviewer, :+1: But there are still some requirements you need to meet before moving to the next milestone. :wave: 

- [ ] Looking at your commits and changes you just made, it seems you created new migration files. Your project now have duplicate migration files for `create_table :users, :posts, :comments, :likes`. This is an issue as it gives error when you run `rails db:migrate` as shown in the screenshots below. I'll suggest you remove these duplicates, drop your database `rails db:drop`, then create and migrate `rails db:create, rails db:migrate`, to make sure everything works well before requesting for another review. :wave: :airplane: 

- - Error screenshot:
![Selection_075](https://github.com/gilberthappi/Blog-app/assets/107632780/665909c6-385c-4e3a-8993-ea117cd0c99b)
![Selection_076](https://github.com/gilberthappi/Blog-app/assets/107632780/250c4101-fa73-4a30-aa8a-427be1d911c3)

Reply:
well  resolved ✔️